### ISSUE TITLE
Got error when wireshark started with wssdl.lua loaded.

### ISSUE THREAD

**Issue by Akagi201** [diacritic/wssdl#11]
```
Lua: Error during loading:
 /Users/akagi201/.config/wireshark/plugins/wssdl.lua:1618: bad argument #1 to 'getupvalue' (function expected, got number)
```

**Comment by Akagi201** [comment]
my lua plugin. I found that, only when I put my plugin to the fold. I will get the error.

```
local wssdl = require 'wssdl'
utlv = wssdl.packet {
    type: byte();
    length: i32();
    data: utf8z();
}

wssdl.dissect {
    udp.port:set {
        [8327] = utlv:proto('utlv', "utun tlv protocol")
    }
}
```

**Comment by Snaipe** [comment]
This is already fixed by f68035c. While waiting for the release, simply declare `utlv` as `local utlv`

**Comment by Akagi201** [comment]
@Snaipe It's my own problem. I have fixed it.

```
utlv = wssdl.packet {
    type: u8();
    length: i32();
    data: byte();
}
```

BTW, this project is awesome.

I will focus on the development on this project. Maybe I can participate in it. :)
